regparser/grammar/unified.py

```diff
@@ -170,3 +170,21 @@ def appendix_section(match):
             | depth1_c).setParseAction(keep_pos).setResultsName(
             "tail", listAllMatches=True)
         + Optional(Suppress(')'))))
+
+internal_cfr_p = (
+    atomic.title
+    + Suppress("CFR")
+    + atomic.part
+    + Suppress('.')
+    + atomic.section
+    + Optional(depth1_p))
+
+multiple_cfr_p = (
```

Comment:
I think we should start putting examples above these expressions to indicate what they are trying to parse. The unit tests help, but it would be nice to have the documentation right there.